### ISSUE TITLE
Moving annotation slow

### ISSUE THREAD

**Issue by tesar-tech** [oxyplot/oxyplot#1496]
I am moving with annotation based on the slider value: 
As you can see, it is super slow and laggy. Blue line is the annotation, it should follow the mouse.

![f8db250c-2449-4795-b64d-1d2eb866c0c5](https://user-images.githubusercontent.com/33880579/77549566-4b02de00-6eb0-11ea-86d8-0a71fcba1ab3.gif)

### Steps to reproduce

Full UWP project with issue is [here](https://github.com/tesar-tech/OxyplotSlowAnnotation).

1. Move with annotation on plot multiple times: 

``` csharp
annotation.X = e.NewValue;//value from slider
annotation.PlotModel.InvalidatePlot(false);
```

Platform:

UWP SDK for 1903

### Expected behaviour

It should be smooth. In presented project there are multiple plots and step frequency is set to .001. So a lot of changes. But even with single plot annotation isn't smooth enough... 

### Actual behaviour

Moving annotation is slow. If I move some element (let say button) by changing margin it is smoother.

**Comment by Jonarw** [comment]
Oxyplot always redraws the whole plot, even if only one single component changed. So depending on how complex your plot is this can take between 'very short' a 'a long time'. Do I understand correctly that you try to update the plot every millisecond? Have you tried to increase this value to, say, 100ms?

**Comment by tesar-tech** [comment]
@Jonarw Yes I tried, and it's naturally better, but doesn't solve my real scenario (this is just demonstration of my issue).

So there is no way how to redraw just the annotation?

**Comment by objorke** [comment]
This is a limitation in this multi-platform & general purpose library. I think the architecture needs to be changed to support partial redraws...

**Comment by tesar-tech** [comment]
I moved the invalidation to timer tick, so it doesn't slow down the slider. And now it's kind of bearable...
![d2f230ff-78c5-468e-8ee7-1741aab56e33](https://user-images.githubusercontent.com/33880579/79683013-46b7af80-8227-11ea-84dd-d86ae0cda599.gif)

(full code in linked repo)

**Comment by dlgcy** [comment]
how to solve this problem?

**Comment by VisualMelon** [comment]
@dlgcy there is no solution. As @objorke said, the architecture of OxyPlot does not lend itself to incremental updates, and no work has been done to facilitate partial redraws. The `OverlayDemo` in the `WpfExamples` project shows how you can use the UI library to present additional graphics outside of OxyPlot pipeline, which may be an option depending on the platform.

**Comment by dlgcy** [comment]
[缓解方案](https://gitee.com/dlgcy/DLGCY_OxyPlotTester/commit/32a9916d1c71eb60f3558cefd9cb092273a5772f)
